pnc_cli/buildconfigurationsets.py

```diff
@@ -301,3 +301,57 @@ def latest_build_set_records_status(id=None, name=None):
     if len(data_json) > 0:
         data_json.sort(key=lambda obj: obj['id'], reverse=True)
         return "Build Config Set Record #" + str(data_json[0]['id']) + ": " + data_json[0]['status']
+
+@arg("-i", "--id", help="ID of the BuildConfigurationSet to build.", type=types.existing_bc_set_id)
+@arg("-n", "--name", help="Name of the BuildConfigurationSet", type=types.existing_bc_set_name)
+@arg("--temporary-build", help="Temporary builds")
+@arg("--timestamp-alignment", help="Enable timestamp alignment for the temporary builds")
+@arg("-f", "--force", help="Force rebuild of all configurations")
+@arg("--id-revisions", nargs="+", help="Specify exact revisions of BuildConfigurations used in set.\n Format: int:int (bcID:revID)", default=None)
+def build_versioned(id=None, name=None, temporary_build=False, timestamp_alignment=False,
```

Comment:
change it so the --id-revisions is optional parameter for the existing build command (don't create new build_versioned commnad)